backend/immediate_cleanup.py

```
@@ -0,0 +1,73 @@
+        # Get stats after cleanup
+        print("\n📊 AFTER CLEANUP:")
+        stats_after = service.get_archive_stats()
+        for key, value in stats_after.items():
+            print(f"  {key}: {value}")
+        
+        print(f"\n✅ CLEANUP COMPLETE!")
+        print(f"📊 Records permanently deleted: {cleaned_count}")
+        
+        return cleaned_count
+        
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)